_i18n/en/general/privacy.md

```diff
@@ -1,57 +1,82 @@
-- Discovery and search services: When opening the Discover screen in the app, it sends a call to Apple to collect podcast suggestions. They may store the query, including country (which by default is the device's country). These suggestions can be disabled by choosing "off" in the region selector. When using the search feature in AntennaPod, the services PodcastIndex.org, Apple Podcasts and fyyd may store your query, including the search terms. Discovery and search queries also include IP address, time, and app name "AntennaPod". After subscribing, the feed is served from the podcast hoster, meaning the discovery/search service is no longer involved.
-- Synchronization services: When enabled via the settings, AntennaPod synchronizes your data. This data may include login credentials, subscribed podcasts, listened episodes, play, pause & favorite actions with date & time stamps and IP address. For more information, see your provider's privacy policy.
-- Google: If you have activated backup & reset in your phone settings (`Settings` » `Backup & Reset` » `Back up my data`), you should be aware that Android itself will periodically save a copy of your phone's data in Google's servers. This backup contains private information, including your WiFi passwords, messages and call history. It may also include data from AntennaPod and from other apps you use. The developers of AntennaPod do not have access to this data. For more information, see [Google's privacy policy](https://policies.google.com).
+#### Podcast hosters
+- When provided in the podcast settings, a host will receive authentication data you provided.
+- Web servers that provide podcast feeds may collect additional data, such as your IP address, access time and what is being accessed. This includes the episodes you are downloading or streaming. Please refer to their respective privacy policies for details. You can view a podcast’s URL by opening the podcast and pressing the info icon. AntennaPod does not allow remote servers to set cookies. If multiple podcasts are hosted on the same server, the server can detect the list of podcasts that you subscribed to and that are hosted on that server. This can happen when publishers use services like FeedBurner, Podtrac, or similar to distribute their feeds.
```

Comment:
Actually it does allow setting cookies but clears them when the app is restarted. Not sure if that's an important distinction to make

Reply:
No, cookies are only sent to the server that created them